modules/core/src/main/scala/org/tessellation/http/routes/DebugTrustRoutes.scala

```diff
@@ -0,0 +1,57 @@
+package org.tessellation.http.routes
+
+import cats.effect.Async
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+
+import org.tessellation.cli.method.ordinalTrustUpdateInterval
+import org.tessellation.domain.trust.storage.TrustStorage
+import org.tessellation.infrastructure.trust.storage.TrustStorage.TrustStore
+import org.tessellation.schema.SnapshotOrdinal
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.{PublicTrust, SnapshotOrdinalPublicTrust}
+import org.tessellation.sdk.domain.gossip.Gossip
+import org.tessellation.security.hex.Hex
+
+import org.http4s._
+import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
+
+final case class DebugTrustRoutes[F[_]: Async](
+  gossip: Gossip[F],
+  trustStorage: TrustStorage[F]
+) extends Http4sDsl[F] {
+  private[routes] val prefixPath = "/debug/trust"
+
+  private val debug: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "latest" =>
+      val output = for {
+        trust <- trustStorage.getTrust
+        current <- trustStorage.getCurrentOrdinalTrust
+        next <- trustStorage.getNextOrdinalTrust
+      } yield TrustStore(trust, current, next)
+
+      output.flatMap(Ok(_))
+
+    case GET -> Root / "ordinalTrustGossip" =>
```

Comment:
it's common not to use `ordinalTrustGossip` in the routes but `ordinal-trust-gossip`. Also, you used `GET` that does some side effect (gossip msgs). IMO it should be a `POST` method.
Consider this: `POST /trust/ordinal`

Reply:
Per the comment by @TheMMaciek , this endpoint has been removed.

Reply:
Per @TheMMaciek 's comment, this has been removed.